File: Android/MeetingManagerAndroid/app/src/main/java/com/microsoft/office365/meetingmgr/Models/ADUser.java
/*
 * Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license.
 * See LICENSE in the project root for license information.
 */
package com.microsoft.office365.meetingmgr.Models;

import com.microsoft.office365.meetingmgr.Utils;

/**
 * Representation of AAD user
 */
public class ADUser {
    public String userPrincipalName;
    public String displayName;
    public String givenName;
    public String surName;

    @Override
    public String toString() {
        if (!Utils.isNullOrEmpty(displayName)) {
            return displayName;
        }
        return userPrincipalName;
    }
}

File: UWP/Meeting_Manager_UWP/MeetingManager/Models/ADUser.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MeetingManager.Models
{
    public class ADUser
    {
        public string userPrincipalName { get; set; }
        public string displayName { get; set; }
        public string givenName { get; set; }
        public string surName { get; set; }

        public override string ToString()
        {
            if (!string.IsNullOrEmpty(displayName))
            {
                return displayName;
            }
            return userPrincipalName;
        }
    }
}
